lib/screens/home_page.dart

```diff
@@ -107,17 +120,7 @@ class _HomePageState extends State<HomePage> {
     return filteredCourses;
   }
 
-  List<Course> filterCoursesByIterests(List interests, List<Course> courses) {
```

Comment:
why did you remove this?

Reply:
It its a duplicate method, misspelt and never used